integr8ly/integreatly-operator
# [WIP] Mgdapi 5728

# Issue link
https://issues.redhat.com/browse/MGDAPI-5728

# What
- update to reconciliation of alert manager secret. Secret has been moved to rhoam ns and reconciliation is no longer triggered by OO but bootstrap stage.

# Verification steps
- Install and uninstall RHOAM from this branch and confirm it works as expected. 
(ensure to run make cluster/prepare/local first to create monitoring stack)


TODO: 5727 branch needs a rebase.
